.circleci/config.yml

```diff
@@ -57,6 +57,11 @@ workflows:
           <<: *build_allways
           requires:
             - build
+          matrix:
+            parameters:
+              # Amazon EKS Kubernetes versions
+              # See https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
+              k8s-version: ["v1.14.9", "v1.15.11", "v1.16.13", "v1.17.9", "v1.18.9"]
```

Comment:
I'd probably left out 1.14 from the list but I'm fine leaving it here as far as it's not troublesome

Reply:
👍 

Providing compatibility for as much of the popular (still used) versions will only benefit the project.

We should probably align the integration tests for all repos with the [docs](https://github.com/kubeless/kubeless#compatibility-matrix-with-kubernetes)?

Reply:
Agree (but it's been a long time since we updated that piece of docs)